### PR TITLE
Feat: Update doc for emea

### PR DESCRIPTION
**Pull request by marcos-albornoz**
Closes VC-26903

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ in PEM format (e.g. /opt/venafi/bundle.pem) and include it using the
 If you are using Venafi as a Service, verify the following:
 
 - The Venafi as a Service REST API at [https://api.venafi.cloud](https://api.venafi.cloud/swagger-ui.html)
-is accessible from the system where Terraform will run.
+or https://api.eu.venafi.cloud(if you have an EU account) is accessible from the system where Terraform will run.
```
Comment (rvelaVenafi) [inline README.md]: need to format the url in the same way as the other one:
```
[https://api.eu.venafi.cloud](https://api.eu.venafi.cloud/swagger-ui.html)
```

File: README.md [hunk 2]
```diff
@@ -173,6 +173,16 @@ for Terraform version 0.11 and below.
    }
    ```
 
+   **Venafi as a Service for EU**:
+
+   ```text
+   provider "venafi" {
+     url     = "https://api.eu.venafi.cloud"
```
Comment (sbanic-venafi) [inline README.md]: Resolve if you are sure we need to use `api.eu.venafi.com` (as opposed to `api.venafi.eu`).

**Review by sbanic-venafi — APPROVED** [review]

**Review by rvelaVenafi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marcos-albornoz) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (marcos-albornoz) [reply README.md]: It was determined we should use `api.venafi.eu`. The PR is updated to that.

**Review by sbanic-venafi — APPROVED** [review]

**Review by luispresuelVenafi — APPROVED** [review]

**Review by rvelaVenafi — APPROVED** [review]

**Pull request merged**